helmfile/values/userCRDs/flux/fluxv2.yaml.gotmpl

```diff
@@ -0,0 +1,77 @@
+{{- if .Values.user.fluxv2.enabled }}
+clusterRoles:
+  crd-controller:
```

Comment:
`crd-controller` and `flux-admin` are basically the same at the moment. 
The only change is if new verbs are added it is picked up automatically. 
But im wondering if we just should merge them :thinking: